tsconfig.json

```diff
@@ -27,8 +27,8 @@
     // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
 
     /* Additional Checks */
-    "noUnusedLocals": true,                /* Report errors on unused locals. */
-    "noUnusedParameters": true,            /* Report errors on unused parameters. */
+    "noUnusedLocals": true                /* Report errors on unused locals. */
+    //"noUnusedParameters": true,            /* Report errors on unused parameters. */

```

Comment:
Another copied comment:

"Please leave this unchanged in your PR. It is perfectly fine to make this change for testing your local env, but we do want these to be enforced in master. Also I think the branch may be out of sync with the clean branch, might want to pull (we aren't using `noUnusedParameters`)"

Reply:
This is the same as the clean branch.

Reply:
Oh wth github... I had this happen to me earlier. It shouldn't show up if it's the same as the upstream...